.github/workflows/main.yaml

```diff
@@ -83,24 +89,25 @@ jobs:
     runs-on: ubuntu-20.04
     needs: build
     steps:
-      - name: Set up Cloud SDK for sabakan
-        uses: google-github-actions/setup-gcloud@v0.2.1
-        with:
-          project_id: ${{ secrets.NECO_TEST_PROJECT_ID }}
-          service_account_key: ${{ secrets.NECO_TEST_SERVICE_ACCOUNT }}
-          export_default_credentials: true
-
-      - name: Set GCP instance name
-        run: echo "INSTANCE_NAME=sabakan-${{ github.run_number }}-$(TZ=Asia/Tokyo date +%H%M%S)" >> $GITHUB_ENV
-
       - name: Checkout
         uses: actions/checkout@v2
 
       - name: Setup golang
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
         with:
           go-version: ${{ env.go-version }}
 
+      - id: auth
+        uses: google-github-actions/auth@v0
+        with:
+          credentials_json: '${{ secrets.NECO_TEST_SERVICE_ACCOUNT }}'
+
+      - name: Set up Cloud SDK for sabakan
+        uses: google-github-actions/setup-gcloud@v0
```

Comment:
@masa213f 
I suppose released versions (v1, v2, ...) are normally well-controlled and it's safe to upgrade their minor versions. However for v0 softwares, there is no such guarantee. So it's rational to pin their minor versions.

BTW the upstream authors encourage to use `v0` tag possibly because they are confident of no breaking changes in future. How do you think about the risk?

Reply:
I understand your opinion as follows. Is it correct?
- If a major version (v1 or more) has been released, we pin the major version only.
- If a major version is not released, we pin the major and minor versions. The patch version is not required.

Even if breaking changes are released, it just breaks CI.
I thought it was no big problem. So I specified the `v0` only.
But your suggestion sounds like a good idea. Which do you like?

Reply:
fixed